packages/types/custom/core.ts

```diff
@@ -587,12 +587,12 @@ export interface MarkerStyle {
 
 export interface ExtendedMasterportalapiMarkers {
   layers: string[]
-  defaultStyle: MarkerStyle
-  hoverStyle: MarkerStyle
-  selectionStyle: MarkerStyle
-  unselectableStyle: MarkerStyle
+  defaultStyle?: MarkerStyle
+  hoverStyle?: MarkerStyle
+  selectionStyle?: MarkerStyle
+  unselectableStyle?: MarkerStyle
   clusterClickZoom?: boolean
-  dispatchOnMapSelect?: string
+  dispatchOnMapSelect?: [string, any]
```

Comment:
```suggestion
  dispatchOnMapSelect?: [string, unknown]
```
While I proposed to use `any`, the lint warning made me think again. `[string, unknown]` seems to be ever-so-slightly better (I actually had to google again and [this](https://stackoverflow.com/questions/51439843/unknown-vs-any#comment128814886_51439843) sums it up pretty well) and is especially nice regarding `packages/core/src/vuePlugins/actions/useExtendedMasterportalapiMarkers/index.ts`, where we can also use the fixed type for the `dispatchOnMapSelect` parameter (I suggest the duplicate definition there is replaced with `...} : ExtendedMasterportalapiMarkers` for the object) that allows us to remove the `// @ts-expect-error | May be one or two elements, no fixed tuple.` comment further down the file.